src/components/NavBar/LogIn/LogIn.js

```diff
@@ -2,29 +2,43 @@ import React, { Component } from 'react';
 import './LogIn.css';
 
 class LogIn extends Component {
+  constructor(props) {
+    super(props);
 
-  componentDidMount() {
-    let username = localStorage.getItem('username');
-    if(username) {
-      document.getElementById('input-username-login').value = username;
-      document.getElementById('remember-username').checked = 'checked';
-      // Refactor using React's `ref` attribute
+    this.rememberChecked = '';
+    this.usernameRemembered = '';
```

Comment:
@MartyTheeMartian How about `rememberUsername`, for consistency?

Reply:
@elimisteve good catch :)

Reply:
@elimisteve Can you merge this PR? Next PR will have that line added.

Reply:
Want to force push?